tests/langchain/test_langchain_model_export.py

```diff
@@ -2383,6 +2383,15 @@ def test_save_load_chain_as_code(chain_model_signature, chain_path, model_config
             model_config=model_config,
         )
 
+    client = mlflow.tracking.MlflowClient()
+    run_id = run.info.run_id
+    assert client.get_run(run_id).data.params == {
```

Comment:
would be nice to see a realistic example that gets flattened :)

Reply:
Added it in the tests/models/test_utils.py code to check it.